test-data/unit/check-string-format.test

```diff
@@ -0,0 +1,20 @@
+[case acceptFstringWithoutSpecs]
+[builtins fixtures/f_string.pyi]
```

Comment:
> the test i added always passes on my machine

Yep, `[builtins ...]` should go after the case body. Now you defined three empty tests with some text in between. (Ough, the README is really a bit confusing regarding this - I'll open a documentation PR to improve the language there).

```
[case acceptFstringWithoutSpecs]
reveal_type(f"{123} {True} {1 + 2} {'foo'}")  # N: Revealed type is "builtins.str"
[builtins fixtures/f_string.pyi]
```